### ISSUE TITLE
Create a standalone page for Contributing

### ISSUE THREAD

**Issue by frankieroberto** [nhsuk/nhsuk.service-manual.prototype-kit.docs#92]
Currently we have an [About page](https://prototype-kit.service-manual.nhs.uk/about) which contains some information about contributing.

I think it might be worth having a standalone page for it, which could then go into a few more details?

It could mention that you could either contribute to the kit itself or the guidance.

It could suggest that both opening issues and pull requests are both valid ways to contribute.

Could briefly say that you'd need a GitHub account, but these are free.

Could mention some more technical stuff for more experienced contributors:

* code style
* running/writing tests
* browsers etc to test on

Could also mention the open source licence and that we'd give credit and thanks in release notes.

**Comment by vickytnz** [comment]
As a starting point I've started writing some things - I'd like to maybe workshop this a bit with a wider group before we put it into code.

My brief is:
* explain the value of contributing
* show that this is community led and in the open
* show that there are different types of contributions right now (and some that we want to do but aren't quite ready)
* explain the ways people can contribute right now and show through the backlog stuff that we're thinking about (this is also a way to incentivise discipline on labelling tickets) 
* have light such contribution options to both let those that are confident with tech go for it but just triage everyone else to the right support channels to get help to start

------

# Contribute to the NHS prototype kit

The NHS prototype kit is a community project supported by the NHS service manual. This means that anyone can contribute if they want to - you do not have to work for NHS England. There are many ways that you can help improve the NHS prototype kit - even if you are not confident with coding.

You can view what the community is working on from our [open project board on Github](https://github.com/orgs/nhsuk/projects/24/views/1) - if you cannot see this board then contact the service manual team and they will tell you.

## Why contribute to the kit

If you contribute to the kit:

* this keeps it up-to-date as an accessible and open way make HTML prototypes of NHS service
* you will become part of an active and supportive community that will help you learn and develop your skills with the kit
* we will credit you in our update notes and other supporting information unless you tell us not to (for example, because you want to stay anonymous)

If you are employed by NHS England, you will usually be able to use examples of contributing to the kit as an objective for your performance reviews - talk to your line manager for more details.

## How you can contribute to the kit 

There are many different ways that you can contribute, depending on your skill and interests. 

Right now you can:
* add or improve documentation for the kit
* do quality assurance or accessibility testing of new features or guidance
* build new features for the NHS prototype kit

> From 2025 on you will also be able to:
> * take part in user research on using the kit (labelled user research in the project board)
> * help do user research on the kit
> * help to run training sessions to install and build a basic prototype
> We will update this page when we are ready to have these types of contributions. If you are interested in helping us start doing these contributions, contact the service manual team


### Add or improve documentation

You can also help improve the guidance for:
* this document site 
* the pages that people view when they download and install the kit

This includes writing in Plain English to the [NHS content guide](https://service-manual.nhs.uk/content) (and [standardising terms that we only use in the kit](https://github.com/nhsuk/nhsuk.service-manual.prototype-kit.docs/blob/main/README.md) and making the structure of the site easy to use. 

Existing things we want to do for guidance are labelled '[documentation](https://github.com/orgs/nhsuk/projects/24/views/1?pane=info&filterQuery=label%3Adocumentation)' on our project board on on the lists of things we want to do for the kit. You can also add your own.

### Do quality assurance or accessibility testing

You can make sure that the code in our kit is robust and accessible by  [stuff here about what to do - if this gets long it may justify its own page. Should also define a label for it for example ‘accessibility’]

Accessibility issues are labelled '[accessibility](https://github.com/orgs/nhsuk/projects/24/views/1?pane=info&filterQuery=label%3Aaccessibility)' on our project board on on the lists of things we want to do for the kit. You can also add your own.


### Fix bugs or build new features ('enhancements') for the NHS prototype kit

You can help:
* Fix bugs in the kit
* Add functionality that is not yet in the NHS prototype kit but is in other prototype kits - for example is in the GOV.UK Prototype kit
* Build features that make itemiser for the NHS prototype kit to use

Existing things we want to do for new features are labelled ['bug'](https://github.com/orgs/nhsuk/projects/24/views/1?pane=info&filterQuery=label%3A"%3Abug%3A+bug") or [‘enhancement’](https://github.com/orgs/nhsuk/projects/24/views/1?pane=info&filterQuery=label%3Aenhancement)' on our project board on on the lists of things we want to do for the kit. You can also add your own.

## What do do if you want to start contributing

If you are confident with code you can create issues with Github.

If you need more support or have questions, you can contact the service manual team or use the slack channel. We have weekly alignment meetings on a Monday afternoon at 4pm which are optional but helpful if you want to check with a person - ask the service manual team for an invite.

**Comment by vickytnz** [comment]
Added contribution page as a light touch version of the Service Manual pages.

This pull request includes:
* the page
* a link in the footer (for now)

![contribution page](https://github.com/user-attachments/assets/2d729599-abdb-4464-ae76-73ec234d6c33)

**Comment by sarawilcox** [comment]
I think this is very good @vickytnz and @frankieroberto . I've made some content changes to the branch - https://github.com/nhsuk/nhsuk.service-manual.prototype-kit.docs/tree/92-create-a-standalone-page-for-contributing. Mostly just cutting it back a bit.

**Comment by vickytnz** [comment]
Excellent - going to merge now :)
